packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -458,8 +458,14 @@ class JITSnapshotter {
       outputPaths.add(isolateSnapshotInstructions);
     }
 
+    // Intel instructions generated by Dart VM can't be serialized yet,
+    // so for now let's create a snapshot with only the data section.
+    final bool supportsAppJit =
```

Comment:
Did you mean to reverse the condition or name variable `doesnotSupportAppJit` ?

Reply:
Thank you, fixed.